frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/EnvironmentAccordionBody.tsx

```diff
@@ -243,75 +252,83 @@ export const EnvironmentAccordionBody = ({
                                 />
                             </StyledListItem>
                         ))}
-                        {strategies.length < 50 || !manyStrategiesPagination ? (
-                            <StyledContentList>
-                                {strategies.map((strategy, index) => (
-                                    <StyledListItem key={strategy.id}>
-                                        {index > 0 ||
-                                        releasePlans.length > 0 ? (
-                                            <StrategySeparator />
-                                        ) : null}
-
-                                        <ProjectEnvironmentStrategyDraggableItem
-                                            strategy={strategy}
-                                            index={index}
-                                            environmentName={
-                                                featureEnvironment.name
-                                            }
-                                            otherEnvironments={
-                                                otherEnvironments
-                                            }
-                                            isDragging={
-                                                dragItem?.id === strategy.id
-                                            }
-                                            onDragStartRef={onDragStartRef}
-                                            onDragOver={onDragOver(strategy.id)}
-                                            onDragEnd={onDragEnd}
-                                        />
-                                    </StyledListItem>
-                                ))}
-                            </StyledContentList>
-                        ) : (
-                            <PaginatedStrategyContainer>
-                                <StyledAlert severity='warning'>
-                                    We noticed you're using a high number of
-                                    activation strategies. To ensure a more
-                                    targeted approach, consider leveraging
-                                    constraints or segments.
-                                </StyledAlert>
+                        <li>
+                            {releasePlans.length > 0 ? (
+                                <StrategySeparator />
+                            ) : null}
```

Comment:
Nest nested lists within list items and add the separator between release plans and strategies to this element instead of the first strategy.